vespa-osgi-testrunner/src/test/java/com/yahoo/vespa/test/samples/WrongBeforeAllTest.java

```diff
@@ -8,7 +8,7 @@
 public class WrongBeforeAllTest {
 
     @BeforeAll
-    void wrong() { }
+    static void wrong() { }
```

Comment:
Yes, this was intentionally wrong. It's not a test we run, but a test that we run to test our test runner :p The assertion is that we indeed _don't_ run the other test methods, because of this error.